lib/active_admin/filters/active_sidebar.rb

```diff
@@ -10,34 +10,7 @@ def initialize
       end
 
       def block
-        -> do
-          active_filters = ActiveAdmin::Filters::Active.new(active_admin_config, assigns[:search])
-          span do
-            if current_scope
-              h4 I18n.t("active_admin.search_status.current_scope"), style: "display: inline"
-              b scope_name(current_scope), class: "current_scope_name", style: "display: inline"
-            end
-            div style: "margin-top: 10px" do
-              h4 I18n.t("active_admin.search_status.current_filters"), style: "margin-bottom: 10px"
```

Comment:
I see that you removed `style:` parameters and the wrapper div. I see how that inline CSS felt hacky but the previous spacing felt a bit more visually pleasant at least to me. Should we restore it and consider cleaning that up separately?

Reply:
Ok, sure )
Fixed and rebased.

Reply:
Thanks!